docs/performance/benchmark.md

```diff
@@ -0,0 +1,23 @@
+# Benchmark
+
+This document explain how to perform a benchmark of approximate nearest neighbor on the Vald cluster, such as [ann-benchmarks](https://github.com/erikbern/ann-benchmarks).
```

Comment:
I think we need a separated document to explain the code bench? (line 6) 
That document will explain the unit bench (code bench) for users 
```suggestion
This document will show and explain the benchmark of approximate nearest neighbor on the Vald cluster, such as [ann-benchmarks](https://github.com/erikbern/ann-benchmarks), and also explain how to perform the ann-benchmark test on you Vald cluster.
```

Reply:
yap.
the main purpose of this doc is what kind of measuring performance in Vald, load test(e2e) and code bench.
And how to use each benchmark tool will be shown in each document, `perfomance/loadtest` and `performance/codebench`.